### PR TITLE
HS-1539: fix liquibase latitude not null constrain issue

### PR DESCRIPTION
**Pull request by freddy61025**
## Description

Fix last PR [HS-1539 ](https://github.com/OpenNMS-Cloud/horizon-stream/pull/1076) cause liquibase error

## Jira link(s)
- https://opennms.atlassian.net/browse/HS-1539

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

**Review by cgorantla — APPROVED** [review]

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml [hunk 1]
```diff
@@ -37,12 +37,8 @@
 
     <changeSet author="bjanssens" id="0.1.0-add-geo-monitoring-location-columns">
         <addColumn tableName="monitoring_location">
-            <column name="latitude" type="DOUBLE">
```
Comment (splatch) [inline inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml:40]: You can't change liquibase changeset once it got merged into develop. Only one way is making a follow up changeset.

**Review by splatch — CHANGES_REQUESTED** [review]

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (freddy61025) [reply inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml:40]: I think it is not yet go into dev cloud yet.


Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset db/changelog/hs-0.1.0/tables/monitoring-location.xml::0.1.0-add-geo-monitoring-location-columns::bjanssens:
     Reason: liquibase.exception.DatabaseException: ERROR: column \"latitude\" of relation \"monitoring_location\" contains null values [Failed SQL: (0) ALTER TABLE inventory.monitoring_location ADD latitude DOUBLE PRECISION NOT NULL]

**Review by splatch — APPROVED** [review]
Please confirm deployment procedure into cloud environment. From my point of view it is looking good.

**Comment by freddy61025** [comment]
> Please confirm deployment procedure into cloud environment. From my point of view it is looking good.

thanks for reminding will do before merge.

**Review by deejgregor — APPROVED** [review]
This change doesn't seem to be applied to dev (which was my assumption because it's crash looping trying to apply this change), so although I'm normally *very* against it, I'm find editing the changeset here. :)

```
citus=> select * from databasechangelog order by dateexecuted desc limit 1;
             id             | author |                       filename                        |        dateexecuted        | orderexecuted | exectype |               md5sum               |       description        | comments | tag | liquibase | contexts | labels | deployment_id
----------------------------+--------+-------------------------------------------------------+----------------------------+---------------+----------+------------------------------------+--------------------------+----------+-----+-----------+----------+--------+---------------
 0.1.0-node-monitored-state | tbigg  | db/changelog/hs-0.1.0/tables/node-monitored-state.xml | 2023-05-17 14:47:59.483733 |            24 | EXECUTED | 8:c084c2f1d3701fbb615f5e2c300aba66 | addColumn tableName=node |          |     | 4.17.0    |          |        | 4334877190
(1 row)
```

**Pull request merged**